test/utilities/apexUtils.ts

```diff
@@ -9,7 +9,11 @@ import { TextEditor } from 'wdio-vscode-service';
 import { runCommandFromCommandPrompt } from './commandPrompt';
 import { pause } from './miscellaneous';
 
-export async function createApexClassWithTest(name: string): Promise<void> {
+export async function createApexClass(
+  name: string,
+  classText: string,
+  breakpoint?: number
+): Promise<void> {
   const workbench = await browser.getWorkbench();
```

Comment:
I think we can avoid pause with `.wait()` on workbench

Reply:
yep, that's in my other pr